### ISSUE TITLE
add support for integration_provider field (Cloud Integration)

### ISSUE THREAD

**Issue by ConstantinElse** [paultyng/go-newrelic#19]
Hi,

Looks like there is no support for `integration_provider` field mentioned [here](https://docs.newrelic.com/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/rest-api-calls-new-relic-infrastructure-alerts)

As result I cannot create an alert for Cloud Integration metric. e.g. for AWS Elastic Load Balancer: 

```json
    {
      "type": "infra_metric",
      "name": "ELB Healthy Host Count",
      "enabled": true,
      "filter": {
        "and": [
          {
            "is": {
              "provider.elbTag_environment": "prod"
            }
          }
        ]
      },
      "id": 6351055,
      "created_at_epoch_millis": 1532626791478,
      "updated_at_epoch_millis": 1532626791530,
      "policy_id": 287891,
      "event_type": "LoadBalancerSample",
      "select_value": "provider.healthyHostCount.Average",
      "comparison": "below",
      "critical_threshold": {
        "value": 1,
        "duration_minutes": 10,
        "time_function": "all"
      },
      "integration_provider": "Elb"
    },
```

**Comment by paultyng** [comment]
released in 2.1.0

**Comment by ConstantinElse** [comment]
Thanks @paultyng
